examples/push/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/config"
+	. "gopkg.in/src-d/go-git.v4/examples"
```

Comment:
oops, you're right, I just saw the period and i thought that was go-check or something... :+1: